bindings/pyroot_experimental/cppyy/CPyCppyy/CMakeLists.txt

```diff
@@ -71,11 +71,6 @@ if("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang" AND CMAKE_CXX_STANDARD GREATER_EQU
     target_compile_options(cppyy PRIVATE -Wno-deprecated-register)
 endif()
 
-# Disables warnings due to new field tp_vectorcall in Python 3.8
-if(NOT MSVC AND ${PYTHON_VERSION_STRING} VERSION_GREATER_EQUAL "3.8")
```

Comment:
To continue my previous comment and be more precise, I'd just keep this setting and not modify all the CPyCppyy files.